src/aliceVision/image/io.cpp

```diff
@@ -440,7 +584,13 @@ void readImage(const std::string& path,
   configSpec.attribute("raw:use_camera_wb", (imageReadOptions.applyWhiteBalance?1:0)); // white balance correction
   // use_camera_matrix: Whether to use the embedded color profile, if it is present: 0=never, 1 (default)=only for DNG files, 3=always
   configSpec.attribute("raw:use_camera_matrix", 3); // use embeded color profile
+
+#if OIIO_VERSION <= (10000 * 2 + 100 * 0 + 8) // OIIO_VERSION <= 2.0.8
```

Comment:
We have `find_package(OpenImageIO 2.0.9 REQUIRED)` in CMakeLists.txt, so this code is not needed.

Reply:
Yes, looks like an error during the rebase.